customizations/ndv2.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -ex
+
+# Place NDv2 topology file under /opt/microsoft/ndv2
+sudo mkdir -p /opt/microsoft/ndv2
```

Comment:
I don't think we need sudo here? Not really an issue, but maybe a bit cleaner to get rid of it.